apps/studio/components/interfaces/Storage/AnalyticsBuckets/index.tsx

```diff
@@ -7,7 +7,7 @@ import { ScaffoldHeader, ScaffoldSection, ScaffoldSectionTitle } from 'component
 import AlertError from 'components/ui/AlertError'
 import { GenericSkeletonLoader } from 'components/ui/ShimmeringLoader'
 import { useAnalyticsBucketsQuery } from 'data/storage/analytics-buckets-query'
-import { Bucket as BucketIcon } from 'icons'
+import { AnalyticsBucket as AnalyticsBucketIcon } from 'icons'
```

Comment:
I’m using this `TypeBucketIcon` pattern to make it clearer in its use below. Otherwise there are a lot of “AnalyticsBucket” things thrown around.